learningequality/ka-lite
# Address #929--don't let restricted users edit their student info.

Title says it all.

Need this within ~24 hours, to release to the IDOC.

Testing:
- in `local_settings.py`, set `CONFIG_PACKAGE="UserRestricted"`
- Make sure that users cannot sign up (old functionality)
- Make sure that users cannot edit their info (new functionality)

@wangguan59 Do you have time to take a look?
